app/controllers/comments/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Comments::LikesController < ApplicationController
+  def create
```

Comment:
http://www.vikingcodeschool.com/unit-10-advanced-forms-and-active-record/redirection-class-methods-namespacing

I highly recommend doing something like in this reading where you just have a before_filter to store a referer.

It's also `request.referer` with one r.

Yes, the proper spelling via-code is incorrect. Not our fault.

Ignore this if you have a request object that has a `referrer` method, I just didn't see where request.referrer is coming from.